src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/PersonTest.kt

```diff
@@ -57,4 +61,47 @@ internal class PersonTest {
     assertThat(person.familyName).isEqualTo("King")
     assertThat(person.dateOfBirth).isEqualTo(dateOfBirth)
   }
+
+  @Test
+  fun `should map common platform defendant to person with additional fields`() {
```

Comment:
Most of this (aliases and addresses) is verified in the integration test, which proves that it works correctly with the incoming message. If we could check the contacts in there as well, we wouldn't need this test